scripts/contract-test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+FKEY="faucet"
+TESTKEY="test"
+
+CHAINID=123
+MONIKER="localbenchmarktestnet"
+
+# remove existing daemon and client
+rm -rf ~/.emint*
```

Comment:
I don't think we want to remove the existing blockchain data?

Reply:
then is it okay to append previous test logs? If so, i'll remove!